manifests/plugin/tasks.pp

```diff
@@ -20,4 +23,15 @@
     enable => true,
     name   => $service,
   }
+  $cron_state = if $automatic_cleanup {
+    present
+  } else {
+    absent
+  }
+  file { 'foreman-tasks-cleanup-cron':
+    ensure  => $cron_state,
+    owner   => 'root',
```

Comment:
Could you add the ```group => 'root'``` as well? Otherwise Puppet will try to derive it based on the file owner of the manifest which could be different. An explicit mode might be a good idea as well.